src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -19,20 +19,23 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Carlos Pereira De Amorim <carlos@shauri.fr>
  */
 class Event extends BaseEvent
 {
     private $subject;
     private $marking;
     private $transition;
     private $workflow;
+    private $context;
 
-    public function __construct(object $subject, Marking $marking, Transition $transition = null, WorkflowInterface $workflow = null)
+    public function __construct(object $subject, Marking $marking, Transition $transition = null, WorkflowInterface $workflow = null, ?array $context = null)
```

Comment:
is context nullability desired? an empty array would work also isnt it, giving us a consistent return type for getContext()

Reply:
this way, I make sure it is backward compatible.
I don't want to break any code already using it

Reply:
but `?array $context = null` vs `array $context = []` makes no difference, right?

Reply:
you're absolutely right!

I changed it.

Thanks !